compiler/lookups.nim

```diff
@@ -343,7 +343,8 @@ proc addDeclAt*(c: PContext; scope: PScope, sym: PSym, info: TLineInfo) =
   if sym.name.s == "_": return
   let conflict = scope.addUniqueSym(sym)
   if conflict != nil:
-    if sym.kind == skModule and conflict.kind == skModule and sym.owner == conflict.owner:
```

Comment:
The owner of a module is its package, so why didn't this check for as is?

Reply:
Because they have the small package for modules within the same project.